user-management/README.adoc

```diff
@@ -11,23 +11,87 @@ other general information.
+- you can assign fined-grained permissions to user through roles,
+- user get a self-service UI to manage an API token with the rights than what permission they have,
+- and finally, an UI is added in Rudder `Administration` menu to help with user managements.
 
-In the xml file, the role attribute is a list of permissions/roles, separated by
-a comma. Each one adds permissions to the user. If one is wrong, or not correctly
-spelled, the user is set to the lowest rights (NoRights), having access only to the
-dashboard and nothing else.
 
-image::docs/images/usermanagement-display.png[]
+== Unitary permissions
 
-WARNING: No Change Request will be generated when changes are made. The plugin is not subject to validation workflow when Change validation plugin is enabled.
-== Pre-defined roles
+Rudder is using a system of fined-grained permissions to gain access to some part of the UI and to be able to
+execute actions through it or APIs.
+
+A permission is composed of a type and a level:
+
+* Type:  Indicates what kind of data will be displayed and/or can be set/updated by the user
+* Level: Access level to be granted on the related type
+
+=== Permission type
+
+Permission types are linked to Rudder functional domain, like "things related to node management and inventories"
+(identified by the permission type `node`) or "configure and view rules" (identified by the permission id `rule`).
+Rudder plugins can bring new permission types, documented in the plugin, like `cve` (for the CVE plugin).
+
+The list of core permission is:
+
+* *administration*: permissions related to Rudder settings, plugin management, user management, etc.
+* *compliance*: permissions to access compliance information (especially with APIs).
+* *configuration*: permission to manage configuration objects. It is a super-set of `rule`, `group`, `directive`, `technique`, `parameter`.
+* *deployer* and *validator*: permission linked to change validation.
+* *deployment*: permission linked to manual policy generation.
+* *directive*: permission to access, create, delete and configure directives.
+* *group*: permission to access, create, delete and configure groups.
+* *node*: permission to accept or refuse nodes, access inventory, change properties or settings.
+* *parameter*: permission to access, create, delete and configure parameters.
+* *rule*: permission to access, create, delete and configure rules.
+* *technique*: permission to access, create, delete and configure techniques.
+* *userAccount*: permission related to user information access or personal API token management.
+
+In addition to these domain-bound permissions, Rudder manage internally two special permissions:
+
+* `no rights` is a special permission that forbid access to everything. It is given when something not expected happens (like permission computation errors, or `user-management` plugin disabled) so that users don't get more rights than they should be allowed.
```

Comment:
```suggestion
* `no rights` is a special permission that forbids access to everything. It is given when something not expected happens (like permission computation errors, or `user-management` plugin disabled) so that users don't get more rights than they should be allowed.
```